modules/tinymce/src/core/main/ts/keyboard/TableNavigation.ts

```diff
@@ -6,7 +6,8 @@
  */
 
 import { Arr, Fun, Optional } from '@ephox/katamari';
-import { Attribute, Insert, SugarElement } from '@ephox/sugar';
+import { CellLocation, CellNavigation, TableLookup } from '@ephox/snooker';
```

Comment:
Anything we do in the DOM model needs to be replicated in RTC eventually. I don't know that having a second table navigation implementation is a good idea.

Reply:
But we don't yet know what that navigation might look like on the rtc side we might even be able to do it though the model instead of the view. I though snooker would be replaced in the alt model with something new. But I guess we can move thing arou d later the breaking change here is to introduce a new dom model file once we have that and it's required we can choose to move things in and out of it and for the integrator it would work the same.

Reply:
I guess I hadn't thought of that, it just seemed like something that snooker was already good at. But maybe it's not very good at it and RTC can do better.

At a minimum, we'll evaluate the bundle size after this has merged and look at whether things like this bloat the core too much. We can, as you say, move things back and forth freely without flagging it as a breaking change.